com.vrcfury.vrcfury/Runtime/VF/Model/Feature.cs

```diff
@@ -649,11 +649,46 @@ public enum HandSign {
     }
     
     [Serializable]
-    public class Gizmo : NewFeatureModel {
+    public class Gizmo : NewFeatureModel
+    {
+        public enum GizmoDrawType
+        {
+            ARROW,
+            TEXT,
+            CUBE,
+            SPHERE,
+            MESH,
+            LINE,
+            ICON
+        }
+        #region General Information
+        /// <summary>
+        /// The type of gizmo to display
+        /// </summary>
+        public GizmoDrawType type;

```

Comment:
this breaks existing gizmos that are using multiple types

Reply:
If i understand it properly there is a upgrade function that gets called, would i be able to use this to upgrade old gizmo versions into multiple components?